python/pyspark/sql/window.py

```diff
@@ -117,7 +120,8 @@ def rowsBetween(start, end):
     def rangeBetween(start, end):
         """
         Creates a :class:`WindowSpec` with the frame boundaries defined,
-        from `start` (inclusive) to `end` (inclusive).
+        from `start` (inclusive) to `end` (inclusive).Range Between considers values
```

Comment:
Likewise missing a period and the operation is not written correctly. Also what does this mean, that it 'considers values'?

Reply:
Thanks @srowen  & @mgaido91 for reviewing the same will update the same